go/dousername.go

```diff
@@ -0,0 +1,77 @@
+package dousername
+
+import (
+	"math/rand"
+	"strings"
+	"time"
+)
+
+func random(arr []string) string {
+	rand.Seed(time.Now().UnixNano())
+	return arr[rand.Intn(len(arr))]
+}
+
+func randomDesc(seaCreaturess []string, randNoun string, descriptors []string, verbs []string) string {
```

Comment:
Why do we pass in `seaCreaturess`, `descriptors` and `verbs` here? Can we just not use the arrays directly?